src/main/java/ch/wisv/areafiftylan/controller/UserRestController.java

```diff
@@ -109,6 +109,68 @@ User getUserById(@PathVariable Long userId) {
         return new ResponseEntity<>(deletedUser, new HttpHeaders(), HttpStatus.OK);
     }
 
+    //////////// PROFILE MAPPINGS //////////////////
```

Comment:
Nicely done in the UserRestController! Way better than a different ProfileRestController.